pallets/msa/src/lib.rs

```diff
@@ -145,6 +145,10 @@ pub mod pallet {
 		/// There are exactly NumberOfBuckets first-key values in PayloadSignatureRegistry.
 		#[pallet::constant]
 		type NumberOfBuckets: Get<u32>;
+
+		/// The maximum number of providers per delegator
+		#[pallet::constant]
+		type MaxNumberOfProvidersPerDelegator: Get<u16>;
```

Comment:
Are we restricting every msa user to only be able to delegate to n number of providers?

Why?

Reply:
No

Reply:
It's for benchmarking primarily.